backend/capellacollab/settings/modelsources/t4c/repositories/routes.py

```diff
@@ -32,40 +33,14 @@
 router = APIRouter()
 
 
-def load_instance_repository(
-    t4c_repository_id: int,
-    db: Session = Depends(get_db),
-    instance: DatabaseT4CInstance = Depends(load_instance),
-) -> tuple[DatabaseT4CInstance, DatabaseT4CRepository]:
-    try:
-        repository = crud.get_t4c_repository(t4c_repository_id, db)
-    except NoResultFound as e:
-        raise HTTPException(
-            404,
-            {
-                "reason": f"Repository with id {t4c_repository_id} was not found."
-            },
-        ) from e
-    if repository.instance != instance:
-        raise HTTPException(
-            409,
-            {
-                "reason": f"Repository {repository.name} is not part of the instance {instance.name}."
-            },
-        )
-    return repository
-
-
 @router.get(
     "/",
     response_model=T4CRepositories,
 )
 def list_t4c_repositories(
     db: Session = Depends(get_db),
-    token: JWTBearer = Depends(JWTBearer()),
-    instance: DatabaseT4CInstance = Depends(load_instance),
+    instance: DatabaseT4CInstance = Depends(get_existing_instance),
 ) -> T4CRepositories:
-    verify_admin(token, db)
```

Comment:
It seems like you removed admin verification from all routes, is that intentional or did you want to add it as a dependency for the entire `APIRouter`?